docs/samples/Microsoft.ML.Samples/Dynamic/OnnxTransform.cs

```diff
@@ -14,7 +14,7 @@ public static void Example()
         {
             // Download the squeeznet image model from ONNX model zoo, version 1.2
```

Comment:
Should it have a note about referencing the Microsoft.ML.Onnx.TestModels nuget?

Reply:
Done.